### PR TITLE
Routing and initial Page setup

### PR DESCRIPTION
**Pull request by ganeshpatro321**
Signed-off-by: ganeshpatro321 <ganeshpatro321@gmail.com>

### CODE REVIEW

**Comment by aquibbaig** [comment]
Great work @ganeshpatro321 ! 🎉

File: dashboard/v1.1/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/build
+/node_modules
```
Comment (aquibbaig) [inline dashboard/v1.1/.eslintignore]: NIT: Just add a newline

**Comment by aquibbaig** [comment]
@ganeshpatro321 `make test-views-v1.1` fails on the branch. Probably update nodejs.

**Comment by ganeshpatro321** [comment]
Updated packages of both the versions. Lets wait for the build.

**Comment by ganeshpatro321** [comment]
@Harkishen-Singh Please check the build error. I have updated the node_js version. Still there are errors showing up.

**Comment by ganeshpatro321** [comment]
Good to go now, build error resolved. @aquibbaig

**Pull request merged**